framework/src/Volo.Abp.AspNetCore.Mvc.UI.Bootstrap/TagHelpers/Popover/AbpPopoverTagHelperService.cs

```diff
@@ -40,6 +45,11 @@ protected virtual void SetDataTriggerIfDismissible(TagHelperContext context, Tag
             if (TagHelper.Dismissible ?? false)
             {
                 output.Attributes.Add("data-trigger", "focus");
+            }
+            // Dismissible has priority over hoverable

```

Comment:
![image](https://user-images.githubusercontent.com/5793291/77291281-f95f2580-6cee-11ea-98a0-cc59f87a5269.png)


They can be used together.

Reply:
Ok, got it covered. However the problem is with the `disabled` attribute.

`disabled` and `dismissible` are not (were not and maybe even should not be?) working together.

Reply:
Ok, got it covered. However the problem is with the `disabled` attribute.

`disabled` and `dismissible` are not (were not and maybe even should not be?) working together.